db/reviews.cds

```diff
@@ -0,0 +1,39 @@
+namespace my.bookshop;
+
+using
+{
+    my.bookshop as my,
+    User,
+    managed,
+    cuid
+} from '@sap/cds/common';
+
+entity Reviews : cuid, managed
+{
+    @cds.odata.ValueList
+    book : Association to my.Books;
+    @Core.Computed
+    reviewer : User;
```

Comment:
Yeah, I guess the reviewer is redundant and we can use the createdBy or modifiedBy User instead.

Reply:
Yeah, I guess the reviewer is redundant and we can use the createdBy or modifiedBy User instead.

Reply:
We can also keep the reviewer and date and remove the managed aspect. Also fine with that, if we want to showcase that you can also use @cds.on.insert yourself. Whatever you like better..

Reply:
I used the managed aspect in the end. It works nicely with the new restrictions in the ReviewService from business perspective. I also changed the Object Page to show modifiedBy and modifiedAt. If the admin now modifies a review of another user, it will show in these fields.

Reply:
Great!